src/modules/form/components/client/addresses/SimpleAddressInput.tsx

```diff
@@ -27,7 +27,11 @@ const SimpleAddressInput: React.FC<Props> = ({
 
   useEffect(() => {
     if (values.length === 0) {
-      onChange([createInitialValue()]);
+      // setTimeout is needed because if the inputs were not previously rendered, they are not yet registered. So the
+      // timeout allows them to register before we change the value here
+      setTimeout(() => {
+        onChange([createInitialValue()]);
+      });
```

Comment:
I think the other way to fix this would be to descend into all the inputs for hidden groups and register them when the form is displayed. Maybe that would be a more robust solution? I think otherwise this problem would occur anytime we show fields and write initial values in a useEffect

Reply:
doesn't actually appear to be registration related. Not totally sure why this doesn't work but dropping for a cleaner defaultValue solution